homework/basic_html_layout/feedback.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<style>
+    .parag1 {
```

Comment:
Can you choose more specific class name?

Reply:
1. Please do not resolve comments which are not fixed.
2. Please use more specific name for the class, like: `checkbox-container`.
3. For this case you don't need to use styles at all. Please check my comments below to do this without styles.